packages/renderer/src/App.svelte

```diff
@@ -224,76 +227,82 @@ window.events?.receive('navigate', (navigationRequest: unknown) => {
-        <Route
-          path="/kubernetes/persistentvolumeclaims/:name/:namespace/*"
-          breadcrumb="Persistent Volume Claim Details"
-          let:meta
-          navigationHint="details">
-          <PVCDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route path="/kubernetes/deployments" breadcrumb="Deployments" navigationHint="root">
-          <DeploymentsList />
-        </Route>
-        <Route
-          path="/kubernetes/deployments/:name/:namespace/*"
-          breadcrumb="Deployment Details"
-          let:meta
-          navigationHint="details">
-          <DeploymentDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route path="/kubernetes/services" breadcrumb="Services" navigationHint="root">
-          <ServicesList />
-        </Route>
-        <Route
-          path="/kubernetes/services/:name/:namespace/*"
-          breadcrumb="Service Details"
-          let:meta
-          navigationHint="details">
-          <ServiceDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route path="/kubernetes/ingressesRoutes" breadcrumb="Ingresses & Routes" navigationHint="root">
-          <IngressesRoutesList />
-        </Route>
-        <Route
-          path="/kubernetes/ingressesRoutes/ingress/:name/:namespace/*"
-          breadcrumb="Ingress Details"
-          let:meta
-          navigationHint="details">
-          <IngressDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route path="/kubernetes/configmapsSecrets" breadcrumb="ConfigMaps & Secrets" navigationHint="root">
-          <ConfigMapSecretList />
-        </Route>
-        <Route
-          path="/kubernetes/configmapsSecrets/configmap/:name/:namespace/*"
-          breadcrumb="ConfigMap Details"
-          let:meta
-          navigationHint="details">
-          <ConfigMapDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route
-          path="/kubernetes/configmapsSecrets/secret/:name/:namespace/*"
-          breadcrumb="Secret Details"
-          let:meta
-          navigationHint="details">
-          <SecretDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
-        <Route
-          path="/kubernetes/ingressesRoutes/route/:name/:namespace/*"
-          breadcrumb="Route Details"
-          let:meta
-          navigationHint="details">
-          <RouteDetails name={decodeURI(meta.params.name)} namespace={decodeURI(meta.params.namespace)} />
-        </Route>
+        {#if $kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR}
+          <Route path="/kubernetes/*" breadcrumb="Kubernetes" navigationHint="root">
+            <KubernetesEmptyPage />
```

Comment:
Would it be possible not to have the navigation menu (Nodes, config etc.) when we are on the empty page ? could be a follow-up PR (not sure if it is already plan)

Reply:
Yes, it should be the next step: https://github.com/containers/podman-desktop/issues/8571#issuecomment-2362916912

Reply:
Perfect ! Thanks :+1: